File: Herencia/ejercicio 4/Java/Comision.java
class TComisionista extends TEmpleado {
    public double base, comision, ventas;
    public TComisionista(String nombre, double base, double comision, double ventas) {
        super(nombre);
        this.base = base;
        this.comision = comision;
        this.ventas = ventas;
    }
    @Override
    public double getSalario() {
        return base + (comision * ventas);
    }
}

File: Herencia/ejercicio 4/Python/Comision.py
from Empleado import TEmpleado
class TComisionista(TEmpleado):
    def __init__(self, nombre, base, comision, ventas):
        super().__init__(nombre)
        self.base = base
        self.comision = comision
        self.ventas = ventas
    def getSalario(self):
        return self.base + self.comision * self.ventas